src/main/java/com/elixr/poc/service/UserService.java

```diff
@@ -0,0 +1,59 @@
+package com.elixr.poc.service;
+
+import com.elixr.poc.data.User;
+import com.elixr.poc.rest.request.UserRequest;
+import com.elixr.poc.rest.response.ErrorResponse;
+import com.elixr.poc.rest.response.UserResponse;
+import com.elixr.poc.rest.validator.UserValidator;
+import lombok.Builder;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+/* service class for  editing response and save or update new user to database */
+@Service
+@Builder
```

Comment:
@Builder annotation is not required to use in the service layer class. Remove the @Builder annotation and try to run your application.

Reply:
I have removed @builder from service class.